python/whylogs/migration/uncompound.py

```diff
@@ -114,7 +114,7 @@ def _uncompound_multimetric(col_name: str, metric_name: str, metric: MultiMetric
 def _uncompound_condition_count(
     col_name: str, metric_name: str, metric: ConditionCountMetric, flags: Optional[FeatureFlags] = None
 ) -> Dict[str, ColumnProfileView]:
-    if not _uncompound_condition_count_feature_flag(flags):
+    if metric.hide_from_profile or not _uncompound_condition_count_feature_flag(flags):
```

Comment:
The request we had for data validators was a non-whylabs use case. The uncompound callstack is not the correct place to remove the metrics because it does not cover the callstacks that write profiles with the local/s3/gcs writers. If we want to consolidate data validators and condition count we probably need to use a different implementation to hide these metrics?

But no objections to this change on its own.